src/Classes/Canvas.ts

```diff
@@ -44,18 +40,42 @@ export class Canvas {
     /**
      * Constructor of canvas manager
      * @param container the parent container
-     * @param tagId the particles container element id
      */
-    constructor(container: Container, tagId: string) {
-        const canvasEl = document.querySelector(`#${tagId} > .${Constants.canvasClass}`) as HTMLCanvasElement;
-
+    constructor(container: Container) {
         this.container = container;
-        this.element = canvasEl;
         this.dimension = {
-            height: canvasEl.offsetHeight,
-            width: canvasEl.offsetWidth,
+            width: 0,
+            height: 0,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'height' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5179066)